.gitignore

```diff
@@ -38,3 +38,5 @@ deployments/demo/openapi/specs
 
 # Built gomobile bindings
 cmd/wallet-sdk-gomobile/bindings
+*.aar
```

Comment:
These should already be excluded with the line above.

Reply:
The line above ignores whatever inside `/bindings/` folder. These new lines ignores binaries in all places inside the repo/project.

Reply:
Why would the binaries be inside the repo otherwise?

Reply:
Do they get downloaded there by the package manager for the reference implementation?

Reply:
To avoid instances like https://github.com/trustbloc/wallet-sdk/pull/43#issuecomment-1300820908 and https://github.com/trustbloc/wallet-sdk/pull/43#issuecomment-1300823243